ambry-store/src/main/java/com/github/ambry/store/StatsBasedCompactionPolicy.java

```diff
@@ -239,7 +238,7 @@ private CostBenefitInfo getCostBenefitInfo(LogSegmentName firstLogSegmentName, L
 
     if (weightOnBenefit) {
       if (totalCost < adjustedTotalCost) {
-        logger.info("getCostBenefitInfo {} cost {} adjusted cost {}",
+        logger.trace("getCostBenefitInfo {} cost {} adjusted cost {}",
```

Comment:
nit: why change it to trace? Does it log too much?

Reply:
yes